### ISSUE TITLE
Immersive Portals is incompatible with Sodium

### ISSUE THREAD

**Issue by nekohacker591** [iPortalTeam/ImmersivePortalsMod#1561]
### Is it a mod compatibility issue?

Yes

### Minecraft version

1.20.1

### Describe the issue

Immersive Portals is incompatible with Sodium

**Comment by AnAnonymousPotato** [comment]
yeah it's currently not compatible with the latest version of Sodium and Iris. I really wanted to play this mod with Distant Horizons

**Comment by spidercraft** [comment]
same lol, because of that immersive portal is unuseable at all
